OD_data_cleaning/calc.tsv

```diff
@@ -0,0 +1,385 @@
+	Sample_ID	calc_gr	calc_volumeloss
```

Comment:
I think this template should work.  We can added additional headers later on for more complex operations like calculating the average and std dev of the growth rate for multiple wells as defined by the user (e..g, a column for "Group_ID")